src/components/Drawer/Drawer.jsx

```diff
@@ -22,8 +22,8 @@ function Drawer() {
       >
         <DrawerIcons name="History" type={'HISTORY'} size={'25px'} />
       </NavLink>
-      <NavLink to="/library">
-        <DrawerIcons name="Library" type={'LIBRARY'} size={'25px'} />
+      <NavLink to="/liked">
+        <DrawerIcons name="Liked Videos" type={'LIKE'} size={'25px'} />
       </NavLink>
       <NavLink to="/watch-layer">
```

Comment:
there is a typo here, also the route is not defined in `App.js`

Reply:
> there is a typo here, also the route is not defined in `App.js`

I have yet to add a watch later feature on the app.